### ISSUE TITLE
Add handy way of creating partials outside templates

### ISSUE THREAD

**Issue by tomwys** [ulope/django-reversetag#1]
It will be very helpful if you can easily create partial url in view and pass it to template as a context.

I use following code to achieve that:

```
class ReversibleConstant(object):
    def __init__(self, var):
        self.var = var

    def resolve(self, context, ignore_failures=False):
        return self.var


def create_partial_url(base, *args, **kwargs):
    base = ReversibleConstant(base)
    args = [ReversibleConstant(arg) for arg in args]
    kwargs = dict((name, ReversibleConstant(value)) for name, value in dict(kwargs).iteritems())
    result = Reversible(base, args, kwargs)
    result.resolve({})
    return result
```

**Comment by ulope** [comment]
I thought about adding a way to pre reverse partials at the view level but found that I never had any real need to do that since you can just as easily (partialy) reverse in the template and store the result in the context.

Do you have a specific use-case where you need this behaviour?

**Comment by tomwys** [comment]
I have generic object list template with pagination. Urls for pagination are constructed in different ways, like: "all/(_page_)", "tag/(_tag_)/(_page_)". If I use your solution, I will have to create additional template for all my views only to create partial urls.

**Comment by ulope** [comment]
Right. That would definitely be useful. I will add something to the next release.
